apps/qubit/modules/digitalobject/templates/_show.php

```diff
@@ -1,9 +1,20 @@
-<div class="digital-object-reference">
+<?php if ($usageType == QubitTerm::CHAPTERS_ID || $usageType == QubitTerm::SUBTITLES_ID): ?>
   <?php if (!empty($accessWarning)): ?>
-    <div class="access-warning">
-      <?php echo $accessWarning ?>
-    </div>
-  <?php else: ?>
-    <?php echo get_component('digitalobject', $showComponent, array('iconOnly' => $iconOnly, 'link' => $link, 'resource' => $resource, 'usageType' => $usageType)) ?>
+      <div class="access-warning">
+        <?php echo $accessWarning ?>
+      </div>
+    <?php else: ?>
+      <?php echo get_component('digitalobject', $showComponent, array('iconOnly' => $iconOnly, 'link' => $link, 'resource' => $resource, 'usageType' => $usageType)) ?>
   <?php endif; ?>
-</div>
+    
+<?php else: ?>
+  <div class="digital-object-reference">
+    <?php if (!empty($accessWarning)): ?>
+      <div class="access-warning">
+        <?php echo $accessWarning ?>
+      </div>
+    <?php else: ?>
+      <?php echo get_component('digitalobject', $showComponent, array('iconOnly' => $iconOnly, 'link' => $link, 'resource' => $resource, 'usageType' => $usageType)) ?>
+    <?php endif; ?>
+  </div>
+<?php endif; ?>
```

Comment:
My main goal was to avoid having a random `div` in the `video` element. Mind you, everything works fine whether it's there or not, but it technically shouldn't be there. I can't say for certain whether this will cause an issue in the future, so I went with the safe, albeit inelegant, approach.

Reply:
This also works. Less lines of code, but you're throwing in an extra conditional to handle each part of the `div`.
```php
<?php if ($usageType != QubitTerm::CHAPTERS_ID && $usageType != QubitTerm::SUBTITLES_ID): ?>
  <div class="digital-object-reference">
<?php endif; ?>
    <?php if (!empty($accessWarning)): ?>
      <div class="access-warning">
        <?php echo $accessWarning ?>
      </div>
    <?php else: ?>
      <?php echo get_component('digitalobject', $showComponent, array('iconOnly' => $iconOnly, 'link' => $link, 'resource' => $resource, 'usageType' => $usageType)) ?>
    <?php endif; ?>
<?php if ($usageType != QubitTerm::CHAPTERS_ID && $usageType != QubitTerm::SUBTITLES_ID): ?>
  </div>
<?php endif; ?>

```

Reply:
I personally don't like this solution as you have to be aware of both for any change and it "breaks" indentation. Not a big deal of course, so I'm good with all the options.

Reply:
Fair. The indentation is weird. I'll stick with my original solution in this case.